test/extension/test.sh

```diff
@@ -227,6 +227,7 @@ function tear_down() {
     sql 'DROP ROLE cdb_testmember_2;'
 
     tear_down_database
+    DATABASE=postgres sql postgres 'DROP ROLE IF EXISTS publicuser';
```

Comment:
Will the tests fail if there are dependencies on `publicuser`? (e.g. for some reason in my local env. other databases are using that role).